onboarding/tests/test_scaffolding.py

```diff
@@ -96,4 +99,30 @@ def test_it_migrates_step_3(self, http_request):
         assert scf.lease_type == "RENT_STABILIZED"
         assert scf.receives_public_assistance is True
 
-        assert session_key_for_step(3) not in http_request
+        assert session_key_for_step(3) not in http_request.session
+
+    def test_it_migrates_rent_history_form_info(self, http_request):
+        from rh.schema import RhFormInfo
+
+        key = RhFormInfo._meta.session_key
+
+        http_request.session[key] = {
+            "address": "123 boop st",
+            "borough": "MANHATTAN",
+            "first_name": "boop",
+            "last_name": "jonze",
```

Comment:
Rent history never creates an account for people on justfix, right? And so doesn't need a preferred name?

Reply:
Yup exactly!